### DISCUSSION TITLE
Entity hider - Hide monsters that are attacking other players

### DISCUSSION THREAD

**Discussion by CookedChicken117** [runelite/runelite#16959] [Ideas]
Hide monsters that are attacking other players so that the non-engaged monsters can be attacked without mis-clicking. 

Good instance for this is, in barrow where semi-safespotting melee brothers is possible but there are usually other players engaging the brothers around and it's hard to do the method when the monsters overlap each other

**Comment by bloodred** [comment]
I find this annoying as well, I think a better solution would be to always have the NPC you are interacting with be the one on top of the stack. That way when you left click a stack of NPCs you will always be clicking yours.
